client/src/components/pages/personalProfile/style.css

```diff
@@ -0,0 +1,23 @@
+.form {
```

Comment:
@AbdallahAmmar96 Like Ahmed suggested, it would be safer if this class name wasn't so generic, or if all of these styles were also wrapped in a .personalProfile class or something like that.  Then there's less chance that these styles bleed into other components.